File: C/unit_5/Bg.c
#include <stdio.h>
int main()
{
    char ch;
    long int num, n = 0, p = 0, z = 0;
    do
    {
        printf("Enter the Number : \n");
        scanf("%d", &num);
        if (num > 0)
            p++;
        if (num < 0)
            n++;
        if (num == 0)
            z++;
        printf("Wanna Continue (y/n)?? \n");
        scanf("%c", &ch);
    } while (ch == 'Y' || ch == 'y');
    printf("Total Positive Number : %d", p);
    printf("Total Negetive Number : %d", n);
    printf("Total Zeoes : %d", z);
    return 0;
}

File: Java/unit_5/Bg.java
package Java.unit_5;

import java.util.Scanner;

public class Bg {
    public static void main(String[] args) {
        char ch;
        long num, n = 0, p = 0, z = 0;
        Scanner sc = new Scanner(System.in);
        do {
            System.out.println("Enter the Number : \n");
            num = sc.nextInt();
            if (num > 0)
                p++;
            if (num < 0)
                n++;
            if (num == 0)
                z++;
            System.out.println("Wanna Continue ?? (y/n)");
            ch = sc.next().charAt(0);
        } while (ch == 'Y' || ch == 'y');
        System.out.println("Total Positive Numbers : " + p);
        System.out.println("Total Positive Numbers : " + n);
        System.out.println("Total Zeroes : " + z);
        sc.close();
    }
}
